RELEASE.md

```diff
@@ -0,0 +1,5 @@
+Release type: patch
+
+- *bug*: Add [lxml](https://lxml.de/) as a project requirement. It is used to
+  parse Atom and RSS feeds. See [Pull Request
+  #49](https://github.com/pelican-plugins/image-process/pull/49)
```

Comment:
You don't need to manually add the link to the corresponding pull request, since that can already be done automatically by AutoPub via the [`append-github-contributor = true`](https://github.com/pelican-plugins/image-process/blob/61a8c7545738c8ffb73f4a9ce9a4936397311bbf/pyproject.toml#L62) configuration option, which I just added to this project in 61a8c75. Example of what it looks like: https://github.com/pelican-plugins/share-post/commit/835af2d52cfebf745082d8934706c92d3dd68aa8#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR9